### PR TITLE
RIA-3876:Add out of country screen in start appeal

### PR DESCRIPTION
**Pull request by divisathyan**
RIA-3876:Add out of country screen in start appeal

### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/RIA-3876

**Does this PR introduce a breaking change?** (check one with "x")
```
[ ] Yes
[ x] No
```

### CODE REVIEW

**Review by makowalski — DISMISSED** [review]
LGTM

File: src/test/java/uk/gov/hmcts/reform/iacaseapi/domain/handlers/presubmit/AppealOutOfCountryHandlerTest.java [hunk 1]
```diff
@@ -0,0 +1,130 @@
...
+        when(callback.getCaseDetails()).thenReturn(caseDetails);
+        when(caseDetails.getCaseData()).thenReturn(asylumCase);
+
+        PreSubmitCallbackResponse<AsylumCase> callbackResponse =
+            appealOutOfCountryHandler.handle(PreSubmitCallbackStage.ABOUT_TO_SUBMIT, callback);
+
+        assertNotNull(callbackResponse);
+        assertEquals(asylumCase, callbackResponse.getData());
+        verify(asylumCase, times(1)).write(APPEAL_OUT_OF_COUNTRY, YesOrNo.NO);
+
```
[the remark this answers is not in the document]
Comment (divisathyan) [reply src/test/java/uk/gov/hmcts/reform/iacaseapi/domain/handlers/presubmit/AppealOutOfCountryHandlerTest.java:71]: There is one more test for that

File: src/functionalTest/resources/scenarios/RIA-3876-out-of-country-screening-start-appeal.json [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+    "uri": "/asylum/ccdAboutToSubmit",
+    "credentials": "LegalRepresentative",
+    "input": {
+      "eventId": "startAppeal",
+      "state": "appealStarted",
+      "caseData": {
+        "template": "minimal-appeal-started.json",
+        "replacements": {
+          "appellantInUk": "No"
+        }
```
[the remark this answers is not in the document]
Comment (divisathyan) [reply src/functionalTest/resources/scenarios/RIA-3876-out-of-country-screening-start-appeal.json:14]: added

**Review by Lee365 — APPROVED** [review]
LGTM

**Review by Srihari-Nakkala — APPROVED** [review]
LGTM

**Review by jkumar-moj — APPROVED** [review]

**Pull request merged**